testwebsite/testwebsite/urls.py

```diff
@@ -16,10 +16,13 @@
 """
 from django.contrib import admin
 from django.urls import include, path
+from django.views.generic import TemplateView
 
 urlpatterns = [
+    path("", TemplateView.as_view(template_name="home.html"), name="home"),
```

Comment:
Just out of curiosity, what is the difference between every path? why do this one have TEmplateView.as_view and other include() ?